src/components/TestToolMenu.js

```diff
@@ -47,6 +47,14 @@ const TestToolMenu = props => (
             />
         </TestToolRow>
 
+        {/* When toggled the app will be forced offline. */}
+        <TestToolRow title="Force offline">
+            <Switch
+                isOn={props.network.shouldForceOffline}
```

Comment:
Getting console error
<img width="356" alt="Screenshot 2022-11-24 at 11 21 03 PM" src="https://user-images.githubusercontent.com/83179295/203843716-8a18639c-4c01-4356-8ba7-38cd758315c1.png">

The value of `shouldForceOffline` is undefined.  

Steps:- 
1. Sign out and sign in 
2. Go to preferences

Reply:
Thanks for catching that!